aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -1126,6 +1127,88 @@ inline void NestedTensor_reshape_copy(
 }
 } // namespace
 
+  /**
+   * Create a buffer tensor that is a view of self
+   *
+   * This serves as the boundary between nested and non nested tensor
+   * view conversions
+   *
+   * @return Returns a new non nested tensor that
+   * aliases the same storage as self
+   */
+Tensor NestedTensor_to_buffer(const Tensor& self) {
+  TORCH_CHECK(self.is_nested(), "Can only create a buffer from Nested Tensor");
```

Comment:
Since this function is only registered for nested key, this can be an internal assert?